Tests/LibWebView/CMakeLists.txt

```diff
@@ -1,7 +1,11 @@
+if (ENABLE_LAGOM_LIBWEB)
```

Comment:
Instead of this, let's put this in the already-existing block for LibWeb tests in Lagom/CMakeLists.txt:

https://github.com/SerenityOS/serenity/blob/a0d8990c821fe33d825754a60700e12140afa5f6/Meta/Lagom/CMakeLists.txt#L661-L663